### PR TITLE
Port LFS functions from old metastore

### LINKED ISSUE #11: Generate LFS pointer files before metadata is uploaded to Github Backend
LFS files (.gitattributes, .lfsconfig, and pointer files) are generated from metadata resource, before uploading to Github backend. 

### Acceptance
* [x]  I have LFS config files generated from a metadata before upload to github backend **Done** See https://github.com/datopian/metastore-lib-js/pull/12/commits/d3e2d20d749ce18b17f8dc19beda37042b5115d0
   * [x]  .gitattributes
   * [x]  .lfsconfig
   * [x] pointer files for each resource saved in data folder
* [x] On upload to github backend, I have the following files as part of the repository **Done** See images below
    * [x]  .gitattributes
   * [x]  .lfsconfig
   * [x] data with corresponding pointer files
 * [x] Documentation is updated to show changes **Done** See https://github.com/datopian/metastore-lib-js/blob/main/README.md

Screen Shot 2021-01-14 at 1 39 55 PM
Screen Shot 2021-01-14 at 1 40 12 PM
Screen Shot 2021-01-14 at 1 40 24 PM
Screen Shot 2021-01-14 at 1 40 36 PM
Screen Shot 2021-01-14 at 1 40 49 PM

### Task
* [x] Port LFS helper files from https://github.com/datopian/metastore-lib/blob/master/metastore/backend/git_lfs_helpers.py
* [x] Port LFS config file generation functions from https://github.com/datopian/metastore-lib/blob/master/metastore/backend/github/storage.py 
* [x] Update create method in github backend to use LFS create function
* [x] Update create method in github backend to upload multiple files at once. 
* [x] Update docstring of methods and documentation in ReadMe

### PR DESCRIPTION
**Pull request by risenW**
This pull fixes #11

### CODE REVIEW

File: src/backend/github.js [hunk 1]
```diff
@@ -24,69 +27,69 @@ class GitHubStorage extends StorageBackend {
...
-          let metadataContent = _prepareJsonFile(metadata)
-
-          return _commitFileToGithub(
-            objectId,
-            'datapackage.json',
-            message || this.defaultCommitMessage,
-            metadataContent,
-            undefined,
-            undefined,
+          let filesToUpload = { metadata: metadata, readMe: readMe }
```
Comment (cotts) [inline src/backend/github.js]: you can use `const` here

File: src/backend/github.js [hunk 2]
```diff
@@ -102,9 +105,14 @@ class GitHubStorage extends StorageBackend {
...
+   * @param {*} objectId : Unique name of the repository
+   * @param {*} branch : (Defaults-main), specific branch to look for repo content
+   */
   async fetch(objectId, branch) {
     return new Promise(async (resolve, reject) => {
-      _getRepo(objectId, branch, this.org, this.token)
+      getRepo(objectId, branch, this.org, this.token)
         .then((repo) => {
           let author = repo.author
           let metadata = repo.metadata
```
Comment (cotts) [inline src/backend/github.js]: you can use destructing over here 
```
const { author, metadata } = repo
```

**Review by cotts — CHANGES_REQUESTED** [review]
Review comments

@risenW  : `git_lfs_helper.js` , `git_api_helper.js` and the test files should be camelCase named instead snake_case

File: src/backend/github.js (hunk 2 above)
Comment (risenW) [reply src/backend/github.js]: True, thanks

**Review by cotts — APPROVED** [review]
LGTM 🥇

**Pull request merged**